python/cuspatial/cuspatial/core/gis.py

```diff
@@ -1,11 +1,14 @@
 # Copyright (c) 2019-2020, NVIDIA CORPORATION.
 
+import numpy as np
```

Comment:
Likewise, does it make sense to lump everything in a file called gis.py? I moved spatial_window out... Did I do the wrong thing?

Reply:
No I think you did the right thing. I left the function in place to make it easier/faster to review.

I'm somewhat ambivalent about the file structure at the moment since everything's re-exported on the top-level `cuspatial` module. But if you or others think there's benefit to splitting these out into separate/more descriptive files as we go, I'd be happy to do that too.